gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java

```diff
@@ -16,32 +16,35 @@
 
 package com.netflix.spinnaker.gate.services;
 
+import com.netflix.spinnaker.gate.config.TaskServiceProperties;
 import com.netflix.spinnaker.gate.services.internal.ClouddriverServiceSelector;
 import com.netflix.spinnaker.gate.services.internal.OrcaServiceSelector;
 import com.netflix.spinnaker.security.AuthenticatedRequest;
 import java.util.*;
+import lombok.Data;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
-@ConfigurationProperties(prefix = "task-service")
+@Data
```

Comment:
Still need `@Data` here?

Reply:
I removed the manual getter/setters for a Data object :)

Reply:
Aaah, yes.